### ISSUE TITLE
Add a way to specify an output folder

### ISSUE THREAD

**Issue by starper** [spacchetti/spago#49]
With `psc-package` you can do something like this:
```
psc-package build -- -o path/to/folder
```
and with `pulp` you can use `--build-path` or `-o` flags. It would be nice if `spago` had something like this.

**Comment by starper** [comment]
Maybe `spago.dhall` could be a good place to specify `purs compile` flags? Something like:
```haskell
let Flags =
  { build-path : Text
  ...
  }
let Config =
  { name : Text
  , dependencies : List Text
  , packages : Packages
  , flags : Flags
  }
```

**Comment by f-f** [comment]
@starper yeah, placing them in the `spago.dhall` sounds like a good idea. Though would this mean also providing a command line option to change this behaviour temporarily?

**Comment by starper** [comment]
@f-f I think the best approach is to implement a way to pass flags using `cli` first and later extend it to work with `spago.dhall` to make it a bit more user friendly.

**Comment by f-f** [comment]
@starper fix up in #66: once it's merged you'd be able to run `spago build -- -o path/to/folder`

**Comment by starper** [comment]
@f-f  Thanks :)
